### PR TITLE
add credits for third party libraries

### PR DESCRIPTION
**Pull request by thomaszurkan-optimizely**

### CODE REVIEW

**Comment by thomaszurkan-optimizely** [comment]
build

**Review by mikeproeng37 — CHANGES_REQUESTED** [review]
Can you include a line about first party code and move the 3rd party deps to `Additional Code` 

https://github.com/optimizely/java-sdk/pull/204/files#diff-04c6e90faac2675aa89e2176d2eec7d8R95

File: README.md [hunk 1]
```diff
@@ -114,3 +114,32 @@ If you choose to build the universal framework yourself, you can do so by runnin
 ### Contributing
 Please see [CONTRIBUTING](CONTRIBUTING.md).
 
+### Credits
+
+First-party code (under OptimizelySDKCore/, OptimizelySDKDatafileManager/, OptimizelySDKEventDispatcher/, OptimizelySDKiOS/, OptimizelySDKShared/, OptimizelySDKTVOS/, OptimizelySDKUniversal/, OptimizelySDKUserProfile/, OptimizelySDKUserProfileService/, ) is copyright Optimizely, Inc. and contributors, licensed under Apache 2.0.
```
Comment (mikeproeng37) [inline README.md]: We no longer have code under `OptimizelySDKUserProfile`. I think we deprecated that in favor of `OptimizelySDKUserProfileService`

**Review by mikeproeng37 — CHANGES_REQUESTED** [review]
Just one last thing and we're good to go

**Review by mikeproeng37 — APPROVED** [review]
lgtm

**Comment by spencerwilson-optimizely** [comment]
Can this be merged?

**Pull request merged**